FinniversKit/Sources/Components/Consent/BannerTransparency/BannerTransparencyView.swift

```diff
@@ -46,6 +46,13 @@ public final class BannerTransparencyView: UIView {
                 return
             }
 
+            switch model.logoType {
+            case .finn:
+                logoImageView.image = UIImage(named: .finnLogo)
```

Comment:
The image should be configurable, so this view should be generic when it comes to logo type and configured for a brand where it is used.